src/Infrastructure/Mesh/tests/ESMC_MBMesh_RedistUTest.C

```diff
@@ -197,27 +197,26 @@ int main(int argc, char *argv[]) {
   //   elem_gids[4] = 6;
   // }
 
-  void *out_mesh;
+  MBMesh *out_mesh;
 
   //----------------------------------------------------------------------------
   //NEX_disable_UTest_Multi_Proc_Only
   // call redist algorithm
-  void *meshv = static_cast<void*> (mesh);
-  MBMesh_createredistnodes(&meshv, &num_node_gids, node_gids, &out_mesh, &rc);
-  // MBMesh_createredistelems(&meshv, &num_elem_gids, elem_gids, &out_mesh, &rc);
+  MBMesh_createredistnodes(&mesh, &num_node_gids, node_gids, &out_mesh, &rc);
```

Comment:
Does this test check the results of the redist itself?

Reply:
Is the mesh redist tested both in this file and src/Infrastructure/Mesh/tests/ESMC_MBMesh_GlueUTest.C?  Do we need both?